modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/test/java/com/liferay/portal/search/tuning/rankings/web/internal/util/RankingResultUtilTest.java

```diff
@@ -51,15 +60,55 @@ public class RankingResultUtilTest extends BaseRankingsWebTestCase {
 	public static final LiferayUnitTestRule liferayUnitTestRule =
 		LiferayUnitTestRule.INSTANCE;
 
+	@BeforeClass
+	public static void setUpClass() {
+		BundleContext bundleContext = SystemBundleUtil.getBundleContext();
+
+		Mockito.when(
+			FrameworkUtil.getBundle(Mockito.any())
+		).thenReturn(
+			bundleContext.getBundle()
+		);
+	}
+
+	@AfterClass
+	public static void tearDownClass() {
+		_frameworkUtilMockedStatic.close();
+	}
+
 	@Before
 	public void setUp() throws Exception {
+		RankingResultUtil rankingResultUtil = new RankingResultUtil();
+
+		Snapshot<DocumentBuilderFactory> documentBuilderFactorySnapshot =
+			Mockito.mock(Snapshot.class);
```

Comment:
@slnn Hmmm maybe I missed this thing in your previous change. Is it possible to NOT mock `Snapshot`? You can register the mocked "_documentBuilderFactory" or "_searchResultInterpreterProvider" using the bundle context, and expect that the snapshot can properly return them.